src/NSwag.SwaggerGeneration.AspNetCore/Processors/OperationTagsProcessor.cs

```diff
@@ -0,0 +1,30 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="OperationTagsProcessor.cs" company="NSwag">
+//     Copyright (c) Rico Suter. All rights reserved.
+// </copyright>
+// <license>https://github.com/NSwag/NSwag/blob/master/LICENSE.md</license>
+// <author>Rico Suter, mail@rsuter.com</author>
+//-----------------------------------------------------------------------
+
+using Microsoft.AspNetCore.Mvc.Controllers;
+using NSwag.SwaggerGeneration.Processors;
+using NSwag.SwaggerGeneration.Processors.Contexts;
+
+namespace NSwag.SwaggerGeneration.AspNetCore.Processors
+{
+    /// <summary>Processes the SwaggerTagsAttribute on the operation method.</summary>
+    public class OperationTagsProcessorNetCore : OperationTagsProcessor
```

Comment:
Rename to AspNetCoreOperationTagsProcessor and move to a new file

How breaking is this change? Usually it should produce the same tags as before, right?

Reply:
Will do.

Reply:
Yes, it should produce the same tags only that we leave it up to the runtime to do the controller naming.
In case the controller name is empty, we still fallback to the base class implementation.

Reply:
I have done the renaming.

PS: I did a hard cast to AspNetCoreOperationProcessorContext because that can be taken for granted. For the ControllerActionDescriptor I didn't because at some time you might decide to add support or documenting page controllers as well and then it could be a PageActionDescriptor instead.